public/about/index.html

```diff
@@ -74,8 +74,49 @@ <h2>About</h2>
   </div>
 </footer>
 </div>
-    <script src="http://localhost:1313/js/script.js"></script>
-  </body>
+  <script>
```

Comment:
This looks like it's repeated verbatim in 3 different places. Can we extract it and use a script.js file? I see that the src depended on the hostname for some reason, but we should be able to get to the path with `/js/script.js`

Reply:
This is in the public folder. The public folder is basically like a dist file that's generated when you build the code. This particular javascript is only in one file in the regular code but that file is a partial template that's imported in other places, so when the public folder is generated, all the imported templates used in different pages are deconstructed to their actual content.

Ideally, the public folder should be in the .gitignore file but when I tried doing that, it didn't work.

Reply:
Right, you can create a js folder within public/ with content and then `/js/script.js` should work. That's what I actually meant before.

Reply:
public/js/script.js already exists with the code, so let's use it please vs repeating the same thing in >1 place